src/measure_theory/measure/vector_measure.lean

```diff
@@ -815,4 +828,93 @@ end
 
 end vector_measure
 
+namespace signed_measure
+
+open vector_measure
+
+open_locale measure_theory
+
+/-- The underlying function for `signed_measure.to_measure`. -/
+def to_measure' (s : signed_measure α) (i : set α) (hi : 0 ≤[i] s)
+  (j : set α) (hj : measurable_set j) : ℝ≥0∞ :=
+@coe ℝ≥0 ℝ≥0∞ _ ⟨s.restrict i j, le_trans (by simp) (hi j hj)⟩
+
+/-- Given a signed measure `s` and a positive measurable set `i`, `to_measure`
+provides the measure, mapping measurable sets `j` to `s (i ∩ j)`. -/
+def to_measure (s : signed_measure α) (i : set α) (hi₁ : measurable_set i) (hi₂ : 0 ≤[i] s) :
```

Comment:
I realized I need the symmetric definition of this for negative sets as well. I will add it once I have cache